colosseum/constants.py

```diff
@@ -13,6 +13,9 @@ def __init__(
         self.validators = validators or []
 
     def validate(self, value):
+        if isinstance(value, OtherProperty):
+            return value
```

Comment:
If possible, I'd rather do this as duck typing rather than an explicit instance check. i.e., can we use the existence of a `value` callable on the value object. This allows anyone to use their own object, rather than .

If we *do* use an instance check, I'd rather it be an abstract base class.

Reply:
>his allows anyone to use their own object, rather than .

Yes, indeed much better and general, updated!